res/docker/configuration.toml

```diff
@@ -87,9 +91,9 @@ Type = "consul"
   Port = 48061
 
 [Binding]
-Type="http"
-# SubscribeTopic="events"
-# PublishTopic="example"
+Type="messagebus"
```

Comment:
Changing this will break BlackBox tests :(

Reply:
Can we have the BBT change it to `http` in console and restart service?

Reply:
Or even better, have a profile just for BBT?

Reply:
Added profile for BB tests.